scene/gui/graph_edit.h

```diff
@@ -184,7 +184,9 @@ class GraphEdit : public Control {
 	GraphEditMinimap *minimap;
 	void _top_layer_input(const Ref<InputEvent> &p_ev);
 
-	bool is_in_hot_zone(const Vector2 &pos, const Vector2 &p_mouse_pos);
+	bool is_in_input_hotzone(GraphNode gn, int i, const Vector2 &p_mouse_pos)
```

Comment:
The convention is to use a `p_` prefix for parameter names (see `p_mouse_pos`). Sometimes the original code is not following this convention, but if you touch it, it's worth making sure you do.

Also `gn` and `i` are not a great parameter name in general, you need to be more verbose.

And `GraphNode` being a node has to be passed by pointer, so `GraphNode *gn` (and this would likely require some changes to the method implementation as well).